### ISSUE TITLE
Improve link styles (a11y)

### ISSUE THREAD

**Issue by aviannamiller** [omeka/theme-freedom#15]
Link to body text contrast should be at least 3:1, it's currently 1.5:1, and 2.2:1 on hover/focus.
